solutions/advanced/izho-12-xor.mdx

```diff
@@ -0,0 +1,150 @@
+---
+id: izho-12-xor
+source: IZhO 2012
+title: XOR
+author: Nathan Katov
+---
+
+## Explanation
+
+Firstly, the original problem can be simplified slightly. In the question, we are asked to find the longest interval that satisfies this inequality.
```

Comment:
```suggestion
Firstly, the original problem can be simplified slightly. In the question, we are asked to find the longest interval that satisfies this inequality:
```

Reply:
See [this article](https://mirror.gutenberg-asso.fr/tex.loria.fr/) for reference on how to punctuate sentences and equations, and be consistent with your use of colons and periods.